CHANGELOG.md

```diff
@@ -25,9 +25,9 @@
   ([#5494](https://github.com/facebook/jest/pull/5494))
 
 ### Chore & Maintenance
-
 * `[filenames]` Standardize file names in root
   ([#5500](https://github.com/facebook/jest/pull/5500))
+* `[filenames]` Standardize files names in "integration-tests" folder ([#5513](https://github.com/facebook/jest/pull/5513))
```

Comment:
should be under `master`

Reply:
good catch! didn't realize it was below a release number. I'll fix